web/styles/subscriptions.css

```diff
@@ -1070,6 +1070,12 @@ div.settings-radio-input-parent {
     }
 }
 
+#stream-advanced-configurations {
+    .dropdown-widget-button {
+        color: hsl(0deg 0% 33%);
```

Comment:
This also should have the CSS for disabled case like we have for "Organization permissions".

Reply:
[Commented above](https://github.com/zulip/zulip/pull/30490#discussion_r1662951886), we don't need.